examples/plot_pp_study.py

```diff
@@ -0,0 +1,362 @@
+mitigator = ExponentiatedGradient(classifier, constraint)
+mitigator.fit(data_without_crimerate, y_true, sensitive_features=lstat_binned)
+y_pred_mitigated = mitigator.predict(data_without_crimerate)
+
+sr_mitigated = MetricFrame(selection_rate, y_true, y_pred_mitigated, sensitive_features=lstat_binned)
+print(sr_mitigated.overall)
+print(sr_mitigated.by_group)
+
+as_mitigated = MetricFrame(accuracy_score, y_true, y_pred_mitigated, sensitive_features=lstat_binned)
+print(as_mitigated.overall)
+print(as_mitigated.by_group)
+# %%
+#
+# By constraining selection rate disparity across LSTAT groups, we notice
+# a significant difference in the accuracy for the 3rd and 4th bins:
+#
+# **Before Mitigation - 95%+ accuracy on both bins**
+#
+#
+# **After Mitigation - 74% and 42% respectively**
+#
+#
+# The significant drop in accuracy indicates that the underlying issue may
+# not be at the model level but rather a reflection of the data (and
+# sociotechnical context itself). In the given data, LSTAT is a reliable
+# predictor of the crime rate threshold condition, which is a reflection
+# of systemic biases leading to this observed trend. This is why accuracy
+# for the higher bins falls off drastically if we try to achieve similar
+# selection rates across LSTAT bins.
+#
+# Problem Area & Sociotechnical Context
+# -------------------------------------
+#
+# The case study above highlights the importance of considering the
+# sociotechnical aspects of tasks traditionally deemed to be solely ML
+# problems. In this scenario, we’ve seen the grave consequences of failing
+# to acknowledge and ensure such considerations are a part of the design
+# and deployment of such ML systems. The result? A feedback loop of
+# discrimination and injustice due to lack of transparency and biased
+# training data. Some key takeaways for this case include:
+#
+#
+# * Algorithms / Algorithmic decision-making systems are not ‘objective’
+# * Lack of transparency and biased historical data leads to discriminatory models
+# * Biased data -> Biased predictions = Feedback loop of discrimination & injustice
+# * **Not all problems/tasks should be framed and approached as an ML task**
+#
+# For more background, check out this resource: `Predictive Policing
+# algorithms
+# <https://www.technologyreview.com/2020/07/17/1005396/predictive-policing-algorithms-racist-dismantled-machine-learning-bias-criminal-justice/>`_
+# are racist (MIT Technology Review)
+#
+# Conclusion
+# ----------
+#
+# The case study on predictive policing and applications of machine
+# learning techniques to making decision-making in the criminal justice
+# system more ‘objective’ reveals that this socio-technical problem
+# requires extensive thoughtful consideration. The findings show how
+# biased data and lack of transparency in models can result in ‘unfair’
+# models that disproportionately oppress marginalized communities -
+# including low income communities as demonstrated in this case.
```

Comment:
And just to recap, the evidence here is that there were differences in accuracy across LSTAT (SES?) bins, and even using a mitigation approach to constrain selection rate disparity across bins did not reduce that disparity, is that right?

Reply:
Correct, though as @romanlutz  pointed out, there might be an (unintended) takeaway that if the accuracy were higher, there wouldn't be an issue. Adding commentary on the fact that the fundamental issue in the scenario is that 'y_true' is a product of historical overpolicing (which is the case for data in this space generally, and not specific to this scenario), and adding the links you suggested could help ensure the reader walks away with the intended takeaway.